pegasus/sites.v3/code.org/views/minecraft_2022_promo.haml

```diff
@@ -0,0 +1,27 @@
+%link{href: "/css/tutorial-promo.css", rel: "stylesheet"}
+
+:ruby
+  mee_image = "/images/mc/fit-1940x620/mc_mee_wide_large_2022.png"
+  mee_title = I18n.t(:hoc2022_mc_mee_title)
+  mee_description = I18n.t(:hoc2022_mc_mee_description)
+  mee_link = "https://aka.ms/HourofCode2022"
+  mee_link_text = I18n.t(:hoc2018_mc_educators_box1_button)
+
+.tutorial-promo-container
+  .img-container
+    %img.background-img{src: mee_image}
+  .col-50.specific-tutorial.right-tutorial{style: "right: 10px; top: 10px; height: initial"}
+    .tutorial-box
+      %h2.specific-tutorial-heading= mee_title
+      .tutorial-details{style: "height: initial"}
+        %p.tutorial-description
+          = mee_description
+      %a{href: mee_link, target: '_blank'}
+        %button.tutorial-button{style: "color: #4d575f; background-color: white; border-color: white; font-size: 14px"}
+          = mee_link_text
+      - if language_dir_class == 'rtl'
```

Comment:
What's the difference between the 'rtl' and 'ltr' pages? Could this conditional be removed?

Reply:
I believe this stands for "right-to-left" and "left-to-right" as it pertains to how different languages are read, so we should keep them.

Reply:
What I don't understand is how we're styling it differently based on right-to-left and left-to-right. The two styles look the same in the conditional?

I think generally it's best practice to let the browser automatically reflow based on direction. See https://github.com/code-dot-org/code-dot-org/pull/44135#discussion_r779106253 for a longer discussion about this.

I'm good to leave this in if it's actually needed (different styling is applied), but if it's not necessary, I'd be in favor of removing.

Reply:
It looks like the left and right absolute positioning is changing:

```
- if language_dir_class == 'rtl'
   %img{style: "position: absolute; top: 10px; **left: 10px**; width: 60px;", src: '/images/mc/new_tag.png'}
- else
   %img{style: "position: absolute; top: 10px; **right: 10px**; width: 60px", src: '/images/mc/new_tag.png'}
```

(Also sorry for codesplaining what rtl and ltr meant when you definitely knew what they were haha! 🤦‍♀️)

Reply:
Oooo I missed the left and right!! Thank you, all good 👍 .

Also codesplaining –– I like that word! Lol